inferno/run-inferno.sh

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env bash
```

Comment:
It might be useful to add a command in our Makefile to run script. In the future, we could also have it depend on another step that sets up test data to run this locally (perhaps as part of a CI test).